examples/kafka/architect.yml

```diff
@@ -3,39 +3,67 @@ name: examples/kafka
 parameters:
   TOPIC_NAME:
     default: architect
+  KAFKA_LISTENERS:
```

Comment:
Not super familiar with kafka. Why do these need to be parameters? 

When would I override the following:
- KAFKA_LISTENERS
- INTERNAL_KAFKA_ADDR
- EXTERNAL_KAFKA_ADDR

Reply:
`KAFKA_LISTENERS` is overridden for local. The other two don't need to be overridden, but they would have been defined in two places otherwise. I guess that doesn't make it appropriate for a parameter though, so I'll update it

Reply:
Yeah setting the values in the debug block might make more sense.